### DISCUSSION TITLE
Add a badge to the activity bar

### DISCUSSION THREAD

**Discussion by flolucut** [microsoft/vscode-discussions#543] [Extension Development QnA] [answered]
Hello all,

i am working on a project where we develop a Extension, and I am working on the UI.
I already added the view container to the contributions but when I run the Extension with the build-in Development-Host of VSCode it doesn't show.

Ignore the title and the Id here.
![image](https://user-images.githubusercontent.com/116207445/228209985-d2114201-c4b7-4f3e-9bac-e2fdaab786c2.png)

And this would be the activitybar in the Development-Host
![image](https://user-images.githubusercontent.com/116207445/228210265-050d9c64-3f75-4ead-9e29-6e68ecd64b7d.png)

**Comment by isidorn** [comment, marked as answer]
I am not 100% sure what is the question here. But I think you want to add a badge to the activity bar. 
This is possible now. I suggest to use the `ViewBadge` API. More details here https://github.com/microsoft/vscode/pull/159679
